demisto_sdk/commands/common/tests/docker_test.py

```diff
@@ -673,7 +673,9 @@ def test_command_is_docker_image_deprecated(self, requests_mock):
                 ],
             )
             docker_image_validator = mock_docker_image_validator()
-            print(docker_image_validator.is_docker_image_deprecated("demisto/aiohttp"))
+            print(  # noqa: T201
```

Comment:
why not a logger?

Reply:
It's a test file. It doesn't need to use the logger.